cmd/thanos/query.go

```diff
@@ -680,6 +692,12 @@ func runQuery(
 				extprom.WrapRegistererWithPrefix("thanos_query_concurrent_", reg),
 				maxConcurrentQueries,
 			),
+			metrics.NewSeriesQueryPerformanceMetricsAggregator(
```

Comment:
I wonder if we could shorten the name a bit (e.g. to `NewSeriesQueryMetricsAggregator`) - since from the params being used we can infer that it aggregate performance metrics - wdyt? :)

Reply:
also for some of the other names that include `Performance`

Reply:
I'm all in for a shorter name that will still be descriptive. Couldn't find myself. I think your suggestion is good! Might even call it just `metricsm.NewQueryPerformanceAggregator`. What do you think, @jessicalins?